src/js/utils/getFormValues.js

```diff
@@ -0,0 +1,16 @@
+const getFormValues = inputs => {
+  const getID = () => {
+    const id = `_${Math.random()
+      .toString(36)
+      .substring(2, 9)}`;
+    return id;
+  };
```

Comment:
A function in a function is wrong for SOLID pattern. This function though little should be in its own separate file and imported